### PR TITLE
Small home page changes

### PR DESCRIPTION
**Pull request by ret2libc**
image
image

I want to add also another demo for rz-pm and FLIRT auto-detection later, i'm working on them.

### CODE REVIEW

**Comment by ITAYC0HEN** [comment]
1. Zigzag of sections is now messed (based on screenshot)
2. I think it's too-thorough and hardcore knowledge for regular users to be on the front page. Can be great as part of a blogpost\announcement but for regular users it doesn't mean a lot (can be concluded with "Significantly improved the experience of using Rizin shell, with precise help, ...."). Also, it doesn't look clear in a gif, will be better as code snippets in the blogpost

**Comment by ret2libc** [comment]
> Zigzag of sections is now messed (based on screenshot)

What do you mean? There are 2 screenshots

**Comment by ITAYC0HEN** [comment]
zigzag

image

**Comment by ret2libc** [comment]
Oh yeah, will fix that!

File: layouts/partials/main-page.html [hunk 1]
```diff
@@ -193,6 +41,33 @@ <h2>Introducing New Projects</h2>
       </p>
     </div>
   </div>
+  <div class="preview-section__feature">
+    <div class="preview-section__feature__description right">
+      <h2>Powerful shell underneath</h2>
+      <p>
+        We re-designed the shell powering Rizin from scratch, leveraging Tree-sitter to define and parse the shell
```
Comment (XVilka) [inline layouts/partials/main-page.html:48]: Maybe worth adding a link to Tree-Sitter site?

Comment (XVilka) [reply layouts/partials/main-page.html:48]: Also "and parse the shell grammar" -> "and parse the grammar" to avoid tautology.

**Review by XVilka — APPROVED** [review]
LGTM

**Comment by ret2libc** [comment]
@ITAYC0HEN btw could you share your rizin settings for the colors you used in the GIFs?

**Comment by ITAYC0HEN** [comment]
Don't think I have it. It's most probably "ayu"

**Comment by ret2libc** [comment]
image
image

**Comment by ret2libc** [comment]
Please review! @thestr4ng3r @XVilka @wargio @kazarmy @ITAYC0HEN

**Review by XVilka — CHANGES_REQUESTED** [review]
Please address my previous comments. Plus, I think we need to use the different color theme and redo all screenshots on this page.

**Comment by ret2libc** [comment]
> Please address my previous comments. Plus, I think we need to use the different color theme and redo all screenshots on this page.

Which color theme should I use? Please provide one so I can include it in the repository and anyone else will be able to do the same gifs/images in the future.

**Comment by XVilka** [comment]
`focus` probably, it's more fitting the site, IMHO. Or `smyck`.

File: layouts/partials/header.html [hunk 2]
```diff
@@ -141,9 +141,6 @@ <h2 class="hero-section__subtitle">{{ .subtitle }}</h2>
                     class="lni lni-book"></i><span>Documentation</span></a>
         </div>
     </div>
-    <div class="hero-section__separator">
```
Comment (ITAYC0HEN) [inline layouts/partials/header.html:144]: The separator is pretty IMO, we can use it for something else, e.g our latest blogpost, or something.
wdyt? Now the website (in terms of design) looks a bit less good

**Comment by wargio** [comment]
i like it

**Review by wargio — APPROVED** [review]

File: layouts/partials/header.html (hunk 2 above)
Comment (ret2libc) [reply layouts/partials/header.html:144]: Sounds like a good idea to me! I will add this.

**Comment by ITAYC0HEN** [comment]
I used some custom Ayu for the theme. Use ayu as well, it will not look the same but it will probably look better :)

**Comment by ret2libc** [comment]
> Why is the target branch add-android-install?

Fixed. I needed it before, not anymore.

**Review by thestr4ng3r — APPROVED** [review]
Adjusting the color theme will enhance it, but either way I think this is good now.

**Review by XVilka — APPROVED** [review]
I fixed it myself and now it's good to go.

**Review by kazarmy — APPROVED** [review]
I'm not the best person to evaluate Rizin's home page but it looks good

**Pull request merged**